Formula/f/flight_sql.rb

```diff
@@ -0,0 +1,46 @@
+class FlightSql < Formula
+  desc "Apache Arrow Flight SQL Server - with DuckDB and SQLite back-ends"
+  homepage "https://arrow.apache.org/docs/format/FlightSql.html"
+  url "https://github.com/voltrondata/flight-sql-server-example.git",
+      tag:      "v1.2.1",
+      revision: "6280bfb7642ac640f6352f01081a7a7b1805a318"
+  license "Apache-2.0"
+
+  depends_on "automake" => :build
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "gflags" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl@3" => :build
+  depends_on "python@3.11" => :build
+  depends_on xcode: :build
+  depends_on :macos
+
+  def install
+    mkdir "build"
+    cd "build" do
+      system "cmake", "-S", "..", *std_cmake_args, "-G", "Ninja"
+      system "cmake", "--build", ".", "--target", "install"
+      bin.install "flight_sql_server"
+      bin.install "flight_sql_client"
+    end
```

Comment:
Please use the more modern syntax that defines the build directory for CMake. You can search for cmake in the repo for examples. There is also no need to set the build generator to ninja I suspect. Just let cmake pick

Reply:
Done - except for the Ninja part.  I found the build was faster with Ninja explicitly chosen...

Reply:
That doesn't matter for Homebrew since it will be a precompiled binary for most people. I do however prefer the reduced complexity of the default build system.